Makefile

```diff
@@ -202,9 +206,8 @@ endif
 install: install-porter install-mixins
 
 install-porter:
-	mkdir -p $(HOME)/.porter
-	cp bin/porter* $(HOME)/.porter/
-	ln -f -s $(HOME)/.porter/porter /usr/local/bin/porter
+	mkdir -p $(PORTER_DIR)
+	cp bin/porter* $(PORTER_DIR)
```

Comment:
Was there an issue with the removed symbolic link command?  Or can we add this back in?

Reply:
Yes, that is Linux only. Does not work on Windows.

Reply:
I've put it back but added an or condition not to fail the script if this link cannot be created.